code/_onclick/hud/radial.dm

```diff
@@ -337,3 +337,19 @@ var/global/list/radial_menus = list()
 	return answer
 
 #define RADIAL_INPUT(user, choices) show_radial_menu(user, user, choices)
+
+/*
+	Helper to make a radial menu button with a name and icon for a given atom.
+*/
+/proc/make_item_radial_menu_button(var/atom/movable/AM, var/name_prefix = "", var/name_suffix = "")
+	var/image/radial_button = image(icon = AM.icon, icon_state = AM.icon_state)
```

Comment:
Possibly more rigorous to do something like:
````
var/image/radial_button = null
radial_button.appearance = AM
radial_button.plane = FLOAT_PLANE
radial_button.layer = FLOAT_LAYER
radial_button.name = etc
````

Reply:
Like that? 

```dm
/proc/make_item_radial_menu_button(var/atom/movable/AM, var/name_prefix = "", var/name_suffix = "")
	var/image/radial_button = image()
	radial_button.appearance = AM
	radial_button.plane = FLOAT_PLANE
	radial_button.layer = FLOAT_LAYER
	radial_button.name = "[name_prefix][AM.name][name_suffix]"
	return radial_button
```

Reply:
That looks correct.